.github/workflows/reusable-tagged-publish.yml

```diff
@@ -36,6 +36,8 @@ jobs:
           skip-cypress-binary: true # publishing doesn't need cypress
       - name: Build packages
         run: yarn build
+      - name: Set liveness package to be public
```

Comment:
Why do we set liveness package to public every time when publishing?

I thought we publish liveness separately?

Reply:
This is only when doing a tagged publish (like we do for @next) so that we ensure that it is published on each tagged release. We don't want to publish liveness on @latest because we are not confident in our automated testing and the smoke testing is too laborious for each release